roles/openshift_health_checker/library/etcdstats.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python
+'''
+Ansible module that determines if the size of paths in an etcd cluster exceed a given limit.
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+
+IMPORT_EXCEPTION = None
+
+try:
+    import etcd
+# pylint: disable=invalid-name
+except ImportError as IMPORT_EXCEPTION:
```

Comment:
Hmm, `IMPORT_EXCEPTION` is invalid here, right?

ALL_CAPS is used for constants, and this is not precisely one...